Taskfile.yml

```diff
@@ -246,14 +246,13 @@ tasks:
     deps: [controller:generate-types]
     dir: "{{.CONTROLLER_ROOT}}"
     sources:
-    - "_apis/**/*_gen.go" # depends on all generated types
+    - "apis/**/*_gen.go" # depends on all generated types
     cmds:
-    - mkdir -p ./_apis
-    - find ./_apis -type f -name "zz_generated.*" -delete
+    - mkdir -p ./apis
+    - find ./apis -type f -name "zz_generated.*" -delete
     - if [ -d "./config/crd/bases" ]; then find "./config/crd/bases" -type f -delete; fi
-    # must CD into _apis as it is ignored by default (this is on purpose to not kill gopls!)
```

Comment:
I think we no longer need to `cd apis` when running controller-gen since it isn't prefixed with `_` and so isn't ignored? So `./...` will match it?